registry/rpc/rpcregistry.go

```diff
@@ -105,26 +79,16 @@ func (r *RPCRegistry) Close() {
 }
 
 func (r *RPCRegistry) IsRegistryReady() bool {
-	if r.registryConn != nil {
-		st, err := r.registryConn.State()
-		if err != nil {
-			log.Fatalf("Unable to get the state of rpc connection: %v", err)
-		}
-		connState := st.String()
-		log.Infof("Registry connection state: %s", connState)
-		if connState != grpcConnectionStateReady {
-			log.Errorf("unable to connect to registry connection state: %s", connState)
-			return false
-		}
-		log.Infof("Getting server status...")
-		status, err := r.Status()
-		if err != nil {
-			log.Errorf("unable to get the status of the registry service %v", err)
-			return false
-		}
-		log.Infof("Status of rpc service: %d, connection state: %s", status, connState)
-		return status == pb.HealthCheckResponse_SERVING && err == nil
-	}
+	// NOTE: checking for readyc doesn't work as expected.
```

Comment:
Yeah right. I need to have a deeper look into this one. Unfortunately checking the readyc doesn't work as expected. If you have a better idea, please let me know. TBH I'm just a beginner of the new API of gRPC v1.0 or etcdv3.